website/public/js/controllers/notificationCtrl.js

```diff
@@ -148,8 +148,9 @@ habitrpg.controller('NotificationCtrl',
     });
 
     // Quest invitation modal
-    $rootScope.$watch('party.quest.key && !party.quest.active && party.quest.members[user._id] == undefined', function(after, before){
-      if (after == before || after != true) return;
+    $rootScope.$watch('user.party.quest.RSVPNeeded == true', function(after, before){
```

Comment:
Figured out the problem. Change the condition to: `party.quest.key && user.party.quest.RSVPNeeded == true`, that'll put the party.quest.key in scope so the directive can load it.